lib/index.js

```diff
@@ -345,19 +345,19 @@ setup_brush = function(graph){
           .attr("transform", function(d, i) { return "translate(0,"+ i * 20 +")"; });
 
       legend.append("rect")
-          .attr("x", width + 20)
```

Comment:
In the new commit, the width for the legend is passed in as legend_width, a property of options.